assets/css/index.css

```diff
@@ -11,4 +11,15 @@
     color: #444;
     margin-top: 1em;
     margin-bottom: 3em;
+}
+
+.index .body-text {
+    font-size: 1.5em;
+    font-weight: 300;
```

Comment:
when you remove `h3`, this font-weight will be unecessary

Reply:
After removing h3, I deleted the font-weight but the text was too fat without it, so I came back with the font-weight and it looks better... should be the standard for regular body text in the website, please take a look.